README.md

```diff
@@ -36,13 +36,27 @@ xedgewarp is available in the AUR as [xedgewarp-git](https://aur.archlinux.org/p
 
 xedgewarp is make-based. Hence, you can clone the git repository and compile and install it via
 
-````
+```
 git clone https://github.com/Airblader/xedgewarp
 cd xedgewarp
 make
 sudo make install
 ```
 
+### Dependencies
+
+xedgewarp depends upon varios X11 libraries, including:
```

Comment:
I don't understand the "bits and pieces". A list of depencies should be just that: a list of dependencies. Can you please rephrase it to leave out the "various" and "bits and pieces" part and just state "xedgewarp has the following dependencies"?